src/pages/docs/octopus-cloud/frequently-asked-questions.md

```diff
@@ -185,7 +185,7 @@ We publish our [release updates](https://octopus.com/whatsnew) and provide a [re
 
 ### What is Octopus Cloud’s uptime SLO?
 
-Octopus Cloud's monthly [uptime SLO](/docs/octopus-cloud/uptime-slo) is 99.5%, measured for the 95% percentile of Cloud instances. This target includes all downtime, including scheduled maintenance.
+Octopus Cloud's monthly [uptime SLO](/docs/octopus-cloud/uptime-slo) is 99.5%, measured for the 95th percentile of Cloud instances. This target includes all downtime, including scheduled maintenance.
```

Comment:
@adrianparker can you please confirm this change is correct?

Reply:
Correct, nice spot :)